massgov/mayflower-patternlab
# DP-4581 Event Listing interactive

This PR makes the event listing interactive organism fully functional with:
- event filters form (in scope: zip code sort)
- dynamic results heading
- dynamic pagination
- dynamic image promo (listing)

### Jira ticket
https://jira.state.ma.us/browse/DP-4581

### To test
1. `git pull` + check out branch
1. cd `styleguide` and run `gulp`
1. visit http://localhost:3000/?p=pages-event-listing
  - [ ] verify that the geo sort works by typing in a location to sort results
  - [ ] verify that you can remove the geosort by clicking either the tag or the clear all link in the results heading
  - [ ] verify that events are sorted by date, then by title once the geosort is removed
  - [ ] verify pagination works

1. browse to location listing https://localhost:3000/?p=pages-location-listing
  - [ ] confirm the test script from #433 still completes successfully

**Styleguide:**
http://mayflower.digital.mass.gov/?p=pages-event-listing
